src/pages/index.js

```diff
@@ -161,9 +152,8 @@ function submitCard({ title, url }) {
       cardListSection.addItem(newCard);
       newCardPopup.close();
     })
-    .catch((err) => {
-      console.error(err);
-    })
+    .catch(console.error)
+
     .finally(() => {
       newCardPopup.setLoading(false, "Create");
```

Comment:
You don't have `setLoading` in your project. Now it's called `renderLoading`. That's why there are errors when I try to submit a form